.circleci/config.yml

```diff
@@ -596,17 +596,18 @@ workflows:
       #         only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
       #       branches:
       #         only: unsigned
-      # - build-ios-simulator:
-          # context: mattermost-mobile-unsigned
-          # requires:
-          #   - test
-          # filters:
-          #   branches:
-          #     only:
-          #       - /^build-\d+$/
-          #       - /^build-ios-\d+$/
-          #       - /^build-ios-beta-\d+$/
-          #       - /^build-ios-sim-\d+$/
+      - build-ios-simulator:
+          context: mattermost-mobile-simulator
+          requires:
+            - test
+          filters:
+            branches:
+              only:
+                - /^build-\d+$/
+                - /^build-ios-\d+$/
+                - /^build-ios-beta-\d+$/
+                - /^build-ios-sim-\d+$/
+                - /^(build|ios)-pr-.*/
```

Comment:
As mentioned in the description, I added PR build so it will be easier for GitLab pipeline to pick up the artifacts than building there.

Reply:
I thought the whole point of the gitlab pipeline was to replace the use of circleCI? Is that not the case anymore?

Reply:
Thanks Eli and Elias! Reverted changes in circleci - https://github.com/mattermost/mattermost-mobile/pull/6854/commits/307922185f059a94133e7e90eaaaa3afe107a2ef